sonar/ext.py

```diff
@@ -105,6 +107,20 @@ def init_config(self, app):
             if k.startswith('SONAR_APP_'):
                 app.config.setdefault(k, getattr(config_sonar, k))
 
+    def init_views(self, app):
+        """Initialize the main flask views."""
+        app.url_map.converters['org_code'] = OrganisationCodeConverter
+
+        @app.route('/<org_code:view>')
+        def index(view):
+            """Homepage."""
+            return render_template('sonar/frontpage.html')
+
+        @app.template_filter()
+        def nl2br(string):
+            r"""Replace \n to <br>."""
+            return string.replace('\n', '<br>')
```

Comment:
Why define the route here and not in the main `views.py`?

Reply:
because `nl2br` si a global filter and not related to the document view only.